pkgs/development/tools/misc/asls/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv
+  , makeWrapper
+  , fetchurl
+  , erlangR22
+}:
+
+stdenv.mkDerivation rec {
+  pname = "asls";
+  version = "0.3.0";
+
+  src = fetchurl {
+    url = "https://github.com/saulecabrera/asls/releases/download/v0.3.0/bin.tar.gz";
+    sha256 = "01d6v79zqw62pkv33km090nyn78zxr0vs3yvnw24ykysvqjyaqd3";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+  buildInputs = [ erlangR22 ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp ./asls $out/bin/asls
+    chmod +x $out/bin/asls
+    wrapProgram $out/bin/asls --prefix PATH ":" ${erlangR22}/bin
```

Comment:
I know approximately nothing about Erlang, but the binary also seems to work fine without wrapping? The shebang is patched. Or does embedded code use things from the path?

Reply:
You're totally right, having `erlangR22` as part of the `buildInputs` is enought.